app/controllers/api/v1/base_controller.rb

```diff
@@ -26,6 +26,6 @@ def internal_server_error(exception)
                else
                  { error: 'Internal Server Error' }
                end
-    render json: response, status: :internal_server_error
+    # render json: response, status: :internal_server_error
```

Comment:
This line of code consistently throws double render errors. For example, if you even define a new route and associated method, but write nothing in it, this line will cause a double render error. I don't know what the cause is yet, but it needs to be commented out until the error is found and resolved, otherwise it's hard for people working on routes and controllers to get anything done.

Reply:
if you are not rendering anything in the controller, rails assumes that you are rendering HTML, so when it gets to this line it throws an error

Reply:
That's not the issue with this line. Even when you render something, like a JSON, this line of code still throws the double render error. However, this method shouldn't even be getting called unless there's actually a 500 error. It gets called even when there isn't a 500 error.

Reply:
I opened a PR to arrive the issue you opened and tagged you in it. Please review, merge to master and to your branch